2024-10-14 21:48:26,181 - werkzeug - INFO -  * Restarting with stat
2024-10-14 21:48:26,419 - werkzeug - WARNING -  * Debugger is active!
2024-10-14 21:48:26,419 - werkzeug - INFO -  * Debugger PIN: 211-549-256
2024-10-14 21:48:45,946 - root - ERROR - login() | TypeError("JwtBuilder.get_token() missing 1 required positional argument: 'self'")
2024-10-14 21:48:45,947 - werkzeug - INFO - 127.0.0.1 - - [14/Oct/2024 21:48:45] "[35m[1mPOST /register HTTP/1.1[0m" 500 -
2024-10-14 21:49:41,774 - werkzeug - INFO -  * Detected change in '/media/utsav/77f97bd8-cb4e-4891-9e00-a700efef6596/Personal_git/my-learnings/my-learnings/micro-service/auth_service/app/service/register_service.py', reloading
2024-10-14 21:49:41,827 - werkzeug - INFO -  * Restarting with stat
2024-10-14 21:49:42,005 - werkzeug - WARNING -  * Debugger is active!
2024-10-14 21:49:42,005 - werkzeug - INFO -  * Debugger PIN: 211-549-256
2024-10-14 21:50:29,231 - werkzeug - INFO -  * Detected change in '/media/utsav/77f97bd8-cb4e-4891-9e00-a700efef6596/Personal_git/my-learnings/my-learnings/micro-service/auth_service/app/service/register_service.py', reloading
2024-10-14 21:50:29,292 - werkzeug - INFO -  * Restarting with stat
2024-10-14 21:50:29,493 - werkzeug - WARNING -  * Debugger is active!
2024-10-14 21:50:29,494 - werkzeug - INFO -  * Debugger PIN: 211-549-256
2024-10-14 21:50:40,555 - werkzeug - INFO -  * Detected change in '/media/utsav/77f97bd8-cb4e-4891-9e00-a700efef6596/Personal_git/my-learnings/my-learnings/micro-service/auth_service/app/controller/main.py', reloading
2024-10-14 21:50:40,619 - werkzeug - INFO -  * Restarting with stat
2024-10-14 21:50:40,820 - werkzeug - WARNING -  * Debugger is active!
2024-10-14 21:50:40,820 - werkzeug - INFO -  * Debugger PIN: 211-549-256
2024-10-14 21:50:46,247 - root - ERROR - register() | TypeError("JwtBuilder.get_token() missing 1 required positional argument: 'self'")
2024-10-14 21:50:46,247 - werkzeug - INFO - 127.0.0.1 - - [14/Oct/2024 21:50:46] "[35m[1mPOST /register HTTP/1.1[0m" 500 -
2024-10-14 21:51:36,087 - werkzeug - INFO -  * Detected change in '/media/utsav/77f97bd8-cb4e-4891-9e00-a700efef6596/Personal_git/my-learnings/my-learnings/micro-service/auth_service/app/service/login_service.py', reloading
2024-10-14 21:51:36,146 - werkzeug - INFO -  * Restarting with stat
2024-10-14 21:51:36,333 - werkzeug - WARNING -  * Debugger is active!
2024-10-14 21:51:36,333 - werkzeug - INFO -  * Debugger PIN: 211-549-256
2024-10-14 21:51:42,367 - werkzeug - INFO -  * Detected change in '/media/utsav/77f97bd8-cb4e-4891-9e00-a700efef6596/Personal_git/my-learnings/my-learnings/micro-service/auth_service/app/service/register_service.py', reloading
2024-10-14 21:51:42,426 - werkzeug - INFO -  * Restarting with stat
2024-10-14 21:51:42,630 - werkzeug - WARNING -  * Debugger is active!
2024-10-14 21:51:42,630 - werkzeug - INFO -  * Debugger PIN: 211-549-256
2024-10-14 21:51:47,472 - root - ERROR - {
    "file": "/media/utsav/77f97bd8-cb4e-4891-9e00-a700efef6596/Personal_git/my-learnings/my-learnings/micro-service/auth_service/app/controller/main.py",
    "line": 26,
    "func": "add_user",
    "exception_type": "<class 'psycopg2.errors.UniqueViolation'>",
    "exception": "duplicate key value violates unique constraint \"user_pkey\"\nDETAIL:  Key (id)=(2) already exists.\n"
}
2024-10-14 21:51:47,484 - werkzeug - INFO - 127.0.0.1 - - [14/Oct/2024 21:51:47] "[35m[1mPOST /register HTTP/1.1[0m" 201 -
2024-10-14 21:51:54,922 - root - ERROR - {
    "file": "/media/utsav/77f97bd8-cb4e-4891-9e00-a700efef6596/Personal_git/my-learnings/my-learnings/micro-service/auth_service/app/controller/main.py",
    "line": 26,
    "func": "add_user",
    "exception_type": "<class 'psycopg2.errors.UniqueViolation'>",
    "exception": "duplicate key value violates unique constraint \"user_pkey\"\nDETAIL:  Key (id)=(2) already exists.\n"
}
2024-10-14 21:51:54,935 - werkzeug - INFO - 127.0.0.1 - - [14/Oct/2024 21:51:54] "[35m[1mPOST /register HTTP/1.1[0m" 201 -
2024-10-14 21:52:50,960 - werkzeug - INFO -  * Detected change in '/media/utsav/77f97bd8-cb4e-4891-9e00-a700efef6596/Personal_git/my-learnings/my-learnings/micro-service/auth_service/app/db/transaction.py', reloading
2024-10-14 21:52:51,019 - werkzeug - INFO -  * Restarting with stat
2024-10-14 21:52:51,238 - werkzeug - WARNING -  * Debugger is active!
2024-10-14 21:52:51,239 - werkzeug - INFO -  * Debugger PIN: 211-549-256
2024-10-14 21:52:57,880 - root - ERROR - {
    "file": "/media/utsav/77f97bd8-cb4e-4891-9e00-a700efef6596/Personal_git/my-learnings/my-learnings/micro-service/auth_service/app/controller/main.py",
    "line": 26,
    "func": "add_user",
    "exception_type": "<class 'psycopg2.errors.UniqueViolation'>",
    "exception": "duplicate key value violates unique constraint \"user_pkey\"\nDETAIL:  Key (id)=(2) already exists.\n"
}
2024-10-14 21:52:57,880 - root - ERROR - add_user() | UniqueViolation('duplicate key value violates unique constraint "user_pkey"\nDETAIL:  Key (id)=(2) already exists.\n')
2024-10-14 21:53:16,435 - werkzeug - INFO -  * Restarting with stat
